modules/qt/hm.nix

```diff
@@ -39,7 +39,9 @@
 
       recommendedStyles = {
         gnome = if config.stylix.polarity == "dark" then "adwaita-dark" else "adwaita";
+        #NOTE: setting a style for kde might not be desirable as user may want to change it via UI
```

Comment:
In that case, should end-users not simply `stylix.targets.qt.enable = false;`:

```suggestion
```

We should provide sensible default values, if possible.

Reply:
This kind of relates to the above point that I don't know if breeze qualifies as a sensible default, as it doesn't really follow the base16/24 concept of stylix.

Reply:
@trueNAHO unfortunately, I always struggle to have a good grasp on what "most" KDE users that also use Nix actually want here. So, I'll just follow any opinion given here.

Reply:
Ok!